Lab2/app/src/main/java/nm/lab2/FeedReaderContract.java

```diff
@@ -0,0 +1,28 @@
+package nm.lab2;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.provider.BaseColumns;
+
+/**
+ * Created by nmohamed on 9/22/2015.
+ */
+//Schema/contract for SQL database
```

Comment:
I would put this comment in the block comment right above it. Just a javadocs convention.